src/node.cc

```diff
@@ -947,17 +947,56 @@ void* ArrayBufferAllocator::Allocate(size_t size) {
   return malloc(size);
 }
 
+static bool domainHasErrorHandler(const Environment* env,
+  const Local<Object>& domain) {
+  HandleScope scope(env->isolate());
+
+  Local<Value> domain_event_listeners_v = domain->Get(env->events_string());
+  if (!domain_event_listeners_v->IsObject())
+    return false;
+
+  Local<Object> domain_event_listeners_o =
+    domain_event_listeners_v->ToObject();
+
+  if (domain_event_listeners_o->IsNull())
+    return false;
+
+  Local<Value> domain_error_listeners_v =
+    domain_event_listeners_o->Get(env->error_string());
```

Comment:
I'm not sure I understand what you mean. Do you mean getting rid of `if (domain_event_listeners_o->IsNull())`?